.github/workflows/ci.yml

```
@@ -186,12 +185,15 @@ jobs:
         id: list-gems
         run: |
           echo "gems=$(find pkg -type f -name '*.gem' -print0 | xargs -0 jq --compact-output --null-input --args '[$ARGS.positional[]]')" >> $GITHUB_OUTPUT
+      - name: Fetch testing OIDC token
+        uses: sigstore-conformance/extremely-dangerous-public-oidc-beacon@d4bd4052866939de30ca438db71d781243d07f61 # main
```

Just cut this 🙂 

ref: https://github.com/sigstore-conformance/extremely-dangerous-public-oidc-beacon/commit/4a8befcc16064dac9e97f210948d226e5c869bdc

```suggestion
        uses: sigstore-conformance/extremely-dangerous-public-oidc-beacon@4a8befcc16064dac9e97f210948d226e5c869bdc # v1.0.0
```